src/content/documentation/develop/manifest-v3-migration-guide.md

```diff
@@ -315,7 +297,7 @@ Move the extension’s CSP to the the manifest.json key to `extension_pages`, li
 
 ### Web Accessible Resources
 
-Web accessible resources are available only to the sites and extensions specified in the manifest. The developer preview supports `resources` and `matches`, but does not support the `extension_ids` and `use_dynamic_url` properties.
+Web accessible resources are available only to the sites and extensions specified in the manifest. Manifest V3 supports `resources` and `matches`, but does not support the `extension_ids` and `use_dynamic_url` properties.
```

Comment:
That is not accurate. It's not MV3 that supports (or not) features but Firefox.

Also, here, Firefox does support `extension_ids` (since 105)

Reply:
Thanks for spotting that one
```suggestion
Web accessible resources are available only to the sites and extensions specified in the manifest. In Manifest V3, Firefox supports the `extension_ids`, `matches`, and `resources` properties to specify the packaged resources you want to make available. Firefox does not support the `use_dynamic_url` property.
```